### ISSUE TITLE
Keyboard shortcut for displaying block as bullets/numbered list/document

### ISSUE THREAD

**Issue by agentydragon** [Roam-Research/issues#336]
**Describe the problem**

If you want to change the display style of a block, you have to use the mouse, which interrupts writing flow for me.

**Feature implementation**

Add a keyboard shortcut to change the display style of the current block.

If you'd like to follow Google Docs, the shortcuts could be:

* Ctrl+Shift+7 = numbered list
* Ctrl+Shift+8 = bulleted list
* ?? = "view as document"

**Comment by pengzhenghao** [comment]
It would also be great if a list can automatically converted to a numbered list if the first line is stared with `1. ` ! Just like what notion or typora does!

**Comment by ninjudd** [comment]
Agreed. Some way to change this with the keyboard would save me a lot of time. Perhaps even just a shortcut to toggle through the three options like `Cmd-Enter` for TODOs.

**Comment by carmensandoval** [comment]
Yes.
